Core/GDCore/IDE/Events/EventsRefactorer.h

```diff
@@ -208,6 +209,19 @@ class GD_CORE_API EventsRefactorer {
                                      gd::String newString,
                                      bool matchCase);
 
+  /**
+   * Replace all occurrences of a gd::String in string events (comments and
+   * groups)
+   *
+   * \return true if something was modified.
+   */
+  static bool ReplaceStringInStringEvent(gd::ObjectsContainer& project,
+                                         gd::ObjectsContainer& layout,
+                                         gd::BaseEvent& stringEvent,

```

Comment:
```suggestion
                                         gd::BaseEvent& event,
```

That's just an event? A "string event" does not really exist as a notion (and should not exist I would say).